src/components/Footer/Footer.scss

```diff
@@ -6,7 +6,7 @@
   margin-top: -1px;
 
   &--with-graphics {
-    padding-top: 40vw;
+    padding-top: 60vw;
```

Comment:
I feel like this is creating way too large of a gap (also a bit hard-coded) but it might be better to handle this in a separate bug fix so I would tentatively revert this. I'll check with William on how we can fix this.

Reply:
Sounds good

Reply:
So... the sun gradient is what made the entire thing so tall. I tried to scale it down a little but it wouldn't look good anymore. The original design had it go over the container as well. 

<!-- <img src="https://user-images.githubusercontent.com/20177171/140851691-9a335796-adb2-4e46-86e5-3ec90a8c04cf.png" width="500px" /> -->

<img src="https://user-images.githubusercontent.com/20177171/140852671-b90011b7-03d5-4fd4-bf51-c6d6fa7dfb50.png" width="500px" />

I think we can't prevent it from extending out of the container, but we could add a `pointer-events: none;` to the graphics svg so it wouldn't block our click events. 

We could also separate the sun gradient to its own svg, but that would still extend out of the container.